### ISSUE TITLE
name conflicting

### ISSUE THREAD

**Issue by JounQin** [rollup/rollup#1505]
reproduction: https://github.com/JounQin/issues

generated codes:
``` js
var getImages = function () {
};

var index = {
  getImages: function getImages() {
    getImages();
  }
};

export default index;
```

Obviously, there is a conflict of `getImages`.

**Comment by pshrmn** [comment]
I believe that this is a duplicate of #1488

**Comment by Rich-Harris** [comment]
Thanks, this is fixed in 0.46.3
